app/src/main/java/com/teamcatchme/calendar_customview/CalendarItemView.kt

```diff
@@ -0,0 +1,65 @@
+package com.teamcatchme.calendar_customview
+
+import android.content.Context
+import android.graphics.*
+import android.util.AttributeSet
+import android.util.Log
+import android.view.View
+import androidx.core.content.res.ResourcesCompat
+import com.teamcatchme.catchmesample.R
+
+class CalendarItemView @JvmOverloads
+constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0,
+    private val date: Int? = null,
+    private val catchuList: Array<Int> = arrayOf(),
+    private val isPrevious: Boolean = false
+) :
```

Comment:
```suggestion
class CalendarItemView @JvmOverloads constructor(
    context: Context,
    attrs: AttributeSet? = null,
    defStyleAttr: Int = 0,
    private val date: Int? = null,
    private val catchuList: Array<Int> = arrayOf(),
    private val isPrevious: Boolean = false
) : View(context, attrs, defStyleAttr) {
```

우선 생성자 형식은 저런데 뷰의 생성자에 저런 인자들이 들어가도 되나 나도 자세히 몰라서 확인해봐야할 듯

Reply:
오키 일단 해보고 우리 리얼 레포에 PR날리기 전까지 나도 확인해보겠음!!
안 돌아가진 않는데 위험할 수는 있을 것 같아